platform/darwin/src/MGLStyle.mm

```diff
@@ -308,7 +308,7 @@ - (void)removeObjectFromLayersAtIndex:(NSUInteger)index
         [NSException raise:NSRangeException
```

Comment:
Doh, I’m blind! 😆 OK, I remember intentionally using `> … - 1` here to parallel the subtraction below. Now that that line below has been simplified, let’s use `>=` instead.